lms/templates/filetree.html

```diff
@@ -7,7 +7,17 @@
       {%- if file.is_folder -%}
         icon hawcons-icon-folder">
       {%- else -%}
-        icon hawcons-icon-document-text hawcons-icon-file-{{ file.path.rsplit('.')[-1] | e }}">
+        icon hawcons-icon-document-text hawcons-icon-file-{{ file.path.rsplit('.')[-1] | e }}" style="padding-left: 0.5em">
+        <div class="file-comments-count">
+          <span class="badge badge-secondary" style="
+          {%- if file.comments_num -%}
+            margin-left: {{ file.indent }}em; background-color: red">
```

Comment:
Use `badge-success` `badge-danger` instead.

Also, try not to use inline styles where possible (use `class` instead).